capybara/selenium/driver.py

```diff
@@ -1,6 +1,7 @@
 import atexit
 from contextlib import contextmanager
 from selenium import webdriver
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
```

Comment:
Almost! This should actually go above the import for `selenium.webdriver.support`.